js/main.js

```diff
@@ -5,20 +5,22 @@ var $tab = document.querySelector('.tab');
 var $view = document.querySelectorAll('.view');
 var $newButton = document.querySelector('.new-button');
 
-$newButton.addEventListener('click', swapHome);
+$newButton.addEventListener('click', swapViews);
 
-$tab.addEventListener('click', swapEntries);
+$tab.addEventListener('click', swapViews);
 
-function swapEntries(event) {
-  $view[0].className = 'view hidden';
-  $view[1].className = 'view';
-  data.view = 'entries';
-}
+function swapViews(event) {
+
+  for (var i = 0; i < $view.length; i++) {
+    if ($view[i].getAttribute('class') === 'view') {
```

Comment:
You should base this off of the value of `data-view`, not based on what view is currently visible. You will need to add a `data-view` attribute to the `a` elements if you haven't already, its value should be the name of the view they are meant to show when they are clicked.